packages/block-editor/src/components/off-canvas-editor/style.scss

```
@@ -1 +1,16 @@
-//Styles for off-canvas editor, remove this line when you add some css to this file!
+.offcanvas-editor-list-view-tree{
```

```suggestion
.offcanvas-editor-list-view-tree {
```

Is your linting working? Unusual that these things are slipping through the linter net.

---

it was working for other things!! I don't know what I might be missing